### DISCUSSION TITLE
Possible to add new mime type to Code notes / blocks?

### DISCUSSION THREAD

**Discussion by Br0kenSilos** [TriliumNext/Trilium#4425] [Q&A] [answered]
First let me say THANK YOU for creating Trilium Notes. I have Trilium running all the time as it helps me keep track of so many aspects of my life. 

I work in IT and I am not a developer. But I do have and write many scripts. Mainly in BATCH but also in bash, powershell, yaml, and json. I really like the code highlighting. But there is no highlighting syntax for batch (that I can find) in Trilium. Is there any way to add syntax highlighting for batch?

**Comment by jwhonce** [comment, marked as answer]
The Readme calls out [CodeMirror](https://github.com/codemirror/CodeMirror) for code support, I suspect Trilium would need support there first.
